src/transformers/tokenization_bert.py

```diff
@@ -395,9 +406,13 @@ def tokenize(self, text, never_split=None):
         orig_tokens = whitespace_tokenize(text)
         split_tokens = []
         for token in orig_tokens:
-            if self.do_lower_case and token not in never_split:
-                token = token.lower()
-                token = self._run_strip_accents(token)
+            if token not in never_split:
+                if self.do_lower_case:
+                    token = token.lower()
+                    if self.strip_accents != False:
```

Comment:
The code quality doesn't like this and would prefer

```suggestion
                    if self.strip_accents is not False:
```

do you mind changing before we merge?

Reply:
Chabged